test/e2e/operator_groups_e2e_test.go

```diff
@@ -340,27 +340,31 @@ var _ = Describe("Operator Group", func() {
 		})
 
 		// validate provided API clusterroles for the operatorgroup
-		adminRole, err := c.KubernetesInterface().RbacV1().ClusterRoles().Get(context.TODO(), operatorGroup.Name+"-admin", metav1.GetOptions{})
-		require.NoError(GinkgoT(), err)
-		adminPolicyRules := []rbacv1.PolicyRule{
-			{Verbs: []string{"*"}, APIGroups: []string{mainCRD.Spec.Group}, Resources: []string{mainCRDPlural}},
-		}
-		require.Equal(GinkgoT(), adminPolicyRules, adminRole.Rules)
-
-		editRole, err := c.KubernetesInterface().RbacV1().ClusterRoles().Get(context.TODO(), operatorGroup.Name+"-edit", metav1.GetOptions{})
+		existingClusterRoleList, err := c.KubernetesInterface().RbacV1().ClusterRoles().List(context.TODO(), metav1.ListOptions{
+			LabelSelector: labels.SelectorFromSet(ownerutil.OwnerLabel(&operatorGroup, "OperatorGroup")).String(),
+		})
 		require.NoError(GinkgoT(), err)
-		editPolicyRules := []rbacv1.PolicyRule{
-			{Verbs: []string{"create", "update", "patch", "delete"}, APIGroups: []string{mainCRD.Spec.Group}, Resources: []string{mainCRDPlural}},
-		}
-		require.Equal(GinkgoT(), editPolicyRules, editRole.Rules)
+		require.Len(GinkgoT(), existingClusterRoleList.Items, 3)
```

Comment:
FWIW try making this fail, the error output is pretty bad. If this is not 3, what went wrong? Can we encode that in the failure message for the dev that breaks this next? What do they need to go fix? It's not clear what is creating the objects - is it test code?